Argo/Types/Decoded/Decoded.swift

```diff
@@ -32,6 +32,21 @@ public extension Decoded {
 
 public extension Decoded {
   /**
+    Convert a `Decoded` type into a `Decoded` `Optional` type.
+
+    This is useful for when a decode operation should be allowed to fail, such
+    as when decoding an optional property.
+
+    - parameter x: A `Decoded` type
+
+    - returns: The `Decoded` type with a `.TypeMismatch` failure converted to
```

Comment:
Is this accurate? This is just forcing _any_ `Decoded` to be `Success` with an optional value now, right? (also, these are lowercased now)